rag/processador_documentos.py

```
@@ -1,76 +1,227 @@
+                    raise DocumentProcessingError("PDF não contém páginas")
+                
+                if len(reader.pages) > self.pdf_page_limit:
+                    raise DocumentProcessingError(
+                        f"PDF excede o limite de {self.pdf_page_limit} páginas."
+                    )
+
+                texto = ''
+                for i, page in enumerate(reader.pages):
+                    try:
+                        texto += page.extract_text() + '\n'
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)